care/facility/migrations/0003_auto_20230614_1048.py

```diff
@@ -8,6 +8,10 @@ class Migration(migrations.Migration):
         ("facility", "0002_auto_20230613_1657"),
     ]
 
+    replaces = [
+        ("facility", "0331_auto_20230130_1652"),
+    ]
+
```

Comment:
Why is this present?

`migrations/003` (this file) deletes PatientSearch model.
`migrations_old/0331` alters meta field of Asset model.

Reply:
@sainak